test/server/lib/DocApi.ts

```diff
@@ -4407,6 +4425,50 @@ function testDocApi() {
         await webhook1();
       });
 
+      it("should not call to a webhook when columns updated are not in columnIds", async () => {  // eslint-disable-line max-len
+        // Create a test document.
+        const ws1 = (await userApi.getOrgWorkspaces('current'))[0].id;
+        const docId = await userApi.newDoc({ name: 'testdoc5' }, ws1);
+        const doc = userApi.getDocAPI(docId);
+        await axios.post(`${serverUrl}/api/docs/${docId}/apply`, [
+          ['ModifyColumn', 'Table1', 'B', { type: 'Bool' }],
+        ], chimpy);
+
+        const webhook = await autoSubscribe('200', docId, {
+          columnIds: 'A', eventTypes: ['add', 'update']
+        });
```

Comment:
Would it be hard to add another test with a second column in `columnIds`?
This would allow to check whether the `;` works as expected.

Reply:
Your right, I will do it

Reply:
:heavy_check_mark: